config/settings/prod.py

```diff
@@ -57,6 +57,16 @@ def get_secret(path):
     }
 }
 
+# Cache
+# -----
+CACHES = {
+    "default": {
+        "BACKEND": "django.core.cache.backends.redis.RedisCache",
+        "LOCATION": f"redis://:{get_secret(os.environ.get('REDIS_PASSWORD_FILE'))}@{os.environ.get('EXTERNAL_URL')}:6379/1",
```

Comment:
should we manage the secret using Vault? (also in dev.py)

Reply:
We only use the Redis token once during setup and don't use it in the code or pass it in API calls. I'm not sure if storing it in Vault provides any benefits. Are we moving all secrets into Vault? Many of our services still use docker-secret, which is a standard practice. Additionally, I'm unsure how services like Tyk read the Vault during setup. It's possible but might require its own ticket and may not be a straightforward fix. I'm unsure.